tests/layer_tests/pytest.ini

```diff
@@ -2,6 +2,7 @@
 markers =
     nightly
     precommit
+    precommit_tf_fe
```

Comment:
```suggestion
```

Reply:
Not needed, all setup is correct already

Reply:
It is not needed for pytest to work, however, without it, around ~130 warnings to add it are raised, possibly masking important warnings and creating around 500l to scroll to output. It shouldn't break or fix anything, just make logs more clear with just 2 short warnings left